scripts/compatibility.py

```diff
@@ -1,14 +1,49 @@
 #!/usr/bin/env python3
 import os
-from opendbc.car.docs import generate_cars_md, get_all_car_docs
+import jinja2
+from opendbc.car.docs import get_all_car_docs, get_all_footnotes, group_by_make
+from opendbc.car.docs_definitions import BaseCarHarness, CarDocs, CarHarness, Column, Device, ExtraCarsColumn, PartType, SupportType
 
 WEB4_DIR = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), ".."))
 VEHICLES_OUT_FILE = os.path.join(WEB4_DIR, "src/lib/vehicles.json")
 HARNESS_PARTS_OUT_FILE = os.path.join(WEB4_DIR, "static/harness-parts.html")
+HARNESS_CONNECTOR_PARTS_OUT_FILE = os.path.join(WEB4_DIR, "src/lib/non-standard-harness-parts.html")
+
+def generate_cars_md(all_car_docs: list[CarDocs], template_fn: str) -> str:
```

Comment:
had to add extra variables for rendering the template (a lot easier than writing it in the jinja template) so I copied the method from opendbc